inst/rmd/h/pandoc/elevate-section-attrs.js

```diff
@@ -0,0 +1,21 @@
+$(function() {
+  $("div.section[class*='level'], section[class*='level']").each(function(i, el) {
+    var $section = $(el);
+    var headers = $section.children().filter(":header");
```

Comment:
You can use the `.first()` method:

```suggestion
    var $header = $section.children().filter(":header").first();
```

so you won't need `$(headers[0])` later.

Reply:
Thanks, done in https://github.com/rstudio/rmarkdown/pull/1732/commits/63a3717c7c5a673d54ed0ce210863bd0b7ef47f9